packages/test/end-to-end-tests/src/test/container.spec.ts

```diff
@@ -18,20 +18,22 @@ import { MockDocumentDeltaConnection } from "@fluid-internal/test-loader-utils";
 import { LocalCodeLoader, TestObjectProvider, LoaderContainerTracker } from "@fluidframework/test-utils";
 import { ensureFluidResolvedUrl } from "@fluidframework/driver-utils";
 import { requestFluidObject } from "@fluidframework/runtime-utils";
-import { ITestDriver } from "@fluidframework/test-driver-definitions";
+import { ITestDriver, ITelemetryBufferedLogger } from "@fluidframework/test-driver-definitions";
```

Comment:
do we actually need to expose ITelemetryBufferedLogger, as the mocha hook should flush, keeping it transparent for the test

Reply:
Yeah I debated about that too... We can discuss and do a follow-up to get rid of that interface on this side if we want.

Reply:
Oh right - we have to leave the interface on the global declare so that the stress test can flush.  If we want to adopt a convention of giving `logger` in mocha files the smaller type, I'm fine with that.  Again, in a future change.  Going to merge this as soon as my test pipeline run finishes.